docs/sources/tempo/release-notes/v2-6.md

```diff
@@ -0,0 +1,265 @@
+
+If you've instrumented your traces for span links, you can use the `link` scope to search for an attribute within a span link. A span link associates one span with one or more other spans. [PRs [3814](https://github.com/grafana/tempo/pull/3814), [3741](https://github.com/grafana/tempo/pull/3741)]
+
+For more information on span links, refer to the [Span Links](https://opentelemetry.io/docs/concepts/signals/traces/#span-links) documentation in the Open Telemetry project.
+
+You can search for an attribute in your link:
+
+![image](insert image)
+
+We’ve also added autocomplete support for `events` and `links`. [PR [PR 3846](https://github.com/grafana/tempo/pull/3846)]
+
+Tempo 2.6 improves TraceQL performance with these updates:
+
+* Performance improvement for `rate() by ()` queries [PR 3719](https://github.com/grafana/tempo/pull/3719)
+* Add caching to query range queries [PR 3796](https://github.com/grafana/tempo/pull/3796)
+* Only stream diffs on metrics queries [PR 3808](https://github.com/grafana/tempo/pull/3808)
+* Tag value lookup use protobuf internally for improved latency [PR 3731](https://github.com/grafana/tempo/pull/3731)
+* TraceQL metrics queries use protobuf internally for improved latency [PR 3745](https://github.com/grafana/tempo/pull/3745)
+* TraceQL search and other endpoints use protobuf internally for improved latency and resource usage [PR 3944](https://github.com/grafana/tempo/pull/3944)
+* Add local disk caching of metrics queries in local-blocks processor [PR 3799](https://github.com/grafana/tempo/pull/3799)
+* Performance improvement for queries using trace-level intrinsics [PR 3920](https://github.com/grafana/tempo/pull/3920)
+* Use multiple goroutines to unmarshal responses in parallel in the query frontend. [PR 3713](https://github.com/grafana/tempo/pull/3713)
+
+### Native histogram support
+
+The metrics-generator can produce native histograms for high-resolution data. [PR 3789](https://github.com/grafana/tempo/pull/3789)
+
+Native histograms are a data type in Prometheus that can produce, store, and query high-resolution histograms of observations. It usually offers higher resolution and more straightforward instrumentation than classic histograms.
+
+To learn more, refer to the [Native histogram](https://grafana.com/docs/tempo/&lt;TEMPO_VERSION>/metrics-generator/#native-histograms) documentation.
+
+### Performance improvements
+
+One of our major improvements in Tempo 2.6 is the reduction of memory usage due to polling improvements.
```

Comment:
@joe-elliott Could you add some info here?  Which PRs help improove the polling? I think there were 3 or 4.

Reply:
I honestly don't have bandwidth atm to add any color here. These are the 3 PRs that impacted this:

* BlockMeta improvements to reduce the size [PR 3950](https://github.com/grafana/tempo/pull/3950) [PR 3951](https://github.com/grafana/tempo/pull/3951) [PR 3952](https://github.com/grafana/tempo/pull/3952)

Reply:
Thank you! That will work. 
```suggestion
One of our major improvements in Tempo 2.6 is the reduction of memory usage due to polling improvements. [PRs [3950](https://github.com/grafana/tempo/pull/3950), [3951](https://github.com/grafana/tempo/pull/3951), [3952](https://github.com/grafana/tempo/pull/3952])
```